mobile/Ihc/screens/PatientSelectScreen.js

```diff
@@ -20,17 +20,39 @@ class PatientSelectScreen extends Component<{}> {
 
     this.state = {
       rows: [],
+      patientsAtTriage: 0, // number of patients at the Triage station
+      patientsAtSoap: 0, // number of patients at the SOAP station
+      patientsAtPharmacy: 0, // number of patients at the Pharmacy station
     };
     this.props.navigator.setOnNavigatorEvent(this.onNavigatorEvent.bind(this));
   }
 
   convertStatusesToRows(statuses) {
     const columnOrder = ['name', 'birthday', 'checkinTime', 'triageCompleted',
-      'doctorCompleted', 'pharmacyCompleted', 'notes', 'patientKey'];
+      'soapCompleted', 'pharmacyCompleted', 'doctorCompleted', 'notes', 'patientKey'];
 
     // Sort statuses by checkin time for now
     statuses.sort( (status1, status2) => status1.checkinTime - status2.checkinTime );
 
+    // Update number of patients at each station based on the assumed workflow:
+    // Triage station -> Soap station -> Pharmacy station
```

Comment:
Are we always allowed to assume this workflow? What if for some reason they do Soap before Triage? Unless there is code that prevents Soap before Triage, it's dangerous to make assumptions like this